src/lib/data/parser/SymbolKind.h

```diff
@@ -3,6 +3,7 @@
 
 enum SymbolKind
 {	 // these values need to be the same as SymbolKind in Java code
+	SYMBOL_UNKNOWN = -1,
```

Comment:
I'm rather against adding this new enum constant just to return it in the `default:` case, because then we would need to handle the new value in the rest of the code base. Please remove this change and see the comments I made at the location where this enum constant would be used.

Reply:
done